docs/deploy-server.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# This script installs starts a dolt server on your Unix compatible computer.
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell. It should be run as root." >&2
+  exit 1
+fi
+
+_() {
+
+install_dolt() {
+  # Install Dolt if it already doesn't exist
+  echo "Installing Dolt..."
+
+  if ! command -v dolt &> /dev/null
+  then
+    sudo bash -c 'curl -L https://github.com/dolthub/dolt/releases/latest/download/install.sh | bash'
+  fi
+}
+
+setup_configs() {
+   # Set up the dolt user along with core dolt configurations
+  echo "Setting up Configurations..."
+  useradd -r -m -d /var/lib/doltdb dolt
```

Comment:
Want to check if this user exists first, err if so

Reply:
Should we make this configurable? Might be annoying if someone runs the script multiple times/.